client/src/components/Layout/Layout.js

```diff
@@ -88,7 +87,11 @@ const Layout = ({ children, pageContext, footer }) => {
       >
         <Head>
           <title>Iteam</title>
-          <link rel="icon" type="image/png" href={imgFavicon.src} />
+          <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+          <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+          <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
```

Comment:
Is this necessary?

Reply:
Yes. The size is picked based on screen resolution and use case.